alternator/streams.cc

```diff
@@ -390,7 +391,18 @@ future<executor::request_return_type> executor::describe_stream(client_state& cl
 
     auto& opts = bs->cdc_options();
 
-    rjson::set(stream_desc, "StreamStatus", rjson::from_string(opts.enabled() ? "ENABLED"s : "DISABLED"s));
+    auto status = "DISABLED";
+
+    if (opts.enabled()) {
+        auto& metadata = service::get_local_storage_service().get_cdc_metadata();
```

Comment:
Globals!
Avi once told me in review that we're in war against globals, so we should not be adding more global variable references. We should be removing them.
I'm passing this info on.